freqtrade/plot/plotting.py

```diff
@@ -33,21 +33,24 @@ def init_plotscript(config):
         pairs = config["pairs"]
     else:
         pairs = config["exchange"]["pair_whitelist"]
+    assert pairs is not None, ('No pairs available in config.')
 
     # Set timerange to use
     timerange = TimeRange.parse_timerange(config.get("timerange"))
 
     tickers = history.load_data(
-        datadir=Path(str(config.get("datadir"))),
+        datadir=Path(config.get("datadir"), config.get("exchange").get("name")),
```

Comment:
This will fail with the latest develop (and master) code - resulting in `user_data/data/binance/binance`.

The fix for this #2199 / #2205 was merged a few days ago to both branches.

Reply:
I modified `freqtrade.data.history.load_pair_history` Warning message to surface this solution.